installer/install-deno.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+os=$(uname -s | tr "[:upper:]" "[:lower:]")
+
+case $os in
+linux)
+  filename="deno-x86_64-unknown-linux-gnu.zip"
+  ;;
+darwin)
+  if [ $(uname -m) == "x86_64" ]; then
```

Comment:
Determin `intel mac` or `m1 mac`.

Reply:
Thank you. Is this a comment for your todo? or explaining for me?

Reply:
oh, sorry for confusion, just explaining for reviewer 😉